### ISSUE TITLE
Prisma db pull updatedAt missing

### ISSUE THREAD

**Issue by torohaifisch** [prisma/prisma#12076]
### Bug description

Using prisma 3.8.1

i have schema.prisma with this model:

model State {
  stateId   Int      @id @default(autoincrement())
  name      String   @db.VarChar(255)
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
}

i run the migrations on the db and then run prisma db pull, the schema.prisma file gets updated but the @updatedAt decorator is missing

### How to reproduce

1.- create a schema with an updated at attribute and @updatedAt decorator
2.- with an empty db set up run npx prisma migrate dev
3- use npx prisma db pull to pull the db schema

### Expected behavior

The expected behavior is that the pulled schema needs to be the same as the schema that was used to generate the database

### Environment & setup

- OS: Ubuntu
- Database: PostgreSQL
- Node.js version: 16

### Prisma Version

```
prisma 3.8.1
```

**Comment by janpio** [comment]
I can not confirm this. I created a new project with a new database, edited the schema to contain your model, then migrated with `npx prisma db push`. Afterwards this happens:

```
C:\Users\Jan\Documents\throwaway\12076  (12076@1.0.0)
λ npx prisma db pull
Prisma schema loaded from prisma\schema.prisma
Environment variables loaded from .env
Datasource "db": PostgreSQL database "coffee_tarsier", schema "public" at "db-provision-postgres23452b4.c8yxynpcltwd.us-east-1.rds.amazonaws.com:5432"

Introspecting based on datasource defined in prisma\schema.prisma …

✔ Introspected 1 model and wrote it into prisma\schema.prisma in 3.90s

*** WARNING ***

These DateTime fields were enriched with `@updatedAt` information taken from the previous Prisma schema.
- Model "State", field: "updatedAt"
```

The `@updatedAt` attribute is still with `State.updatedAt`.

Am I missing something?

**Comment by janpio** [comment]
Ok, then this is just a misunderstanding: `@updatedAt` is not present in the database at all, it is Prisma side functionality that is not persisted in the database. (We are aware there are some databases where it would be possible to represent this functionality in the database, but the current implementation of `@updatedAt` does not use it.)

**Comment by torohaifisch** [comment]
ah my bad i understand now, i was trying to set up a new project and pull the schema from an existing database but encountered this, so it seems i need to fill those missing decorators myself.

Is there a better option to get a full working prisma schema from the database? because with those missing @updatedAt when i do a create request or something like that prisma raises an error about that missing attribute.

**Comment by janpio** [comment]
No, if the database has no information about it, we can not otherwise get the information besides an existing Prisma schema file. We might revisit the implementation one day in the future, but right now it is not a priority as this works well enough usually. As soon as you have the attribute in your schema, it stays there.

> because with those missing @updatedAt when i do a create request or something like that prisma raises an error about that missing attribute.

Can you elaborate on this?
